src/dl_util.cpp

```diff
@@ -139,8 +139,8 @@ dl_error_t dl_util_load_from_stream( dl_ctx_t dl_ctx,       	dl_typeid_t
 			return DL_ERROR_INTERNAL_ERROR;
 	}
 
-	error = dl_instance_load( dl_ctx, type, load_instance, load_size, load_instance, load_size, 0x0 );
-
```

Comment:
Not really sure if I like this as this now asume dl_util having knowleage of the internals of dl_load*. The thought about dl util is that it should only use the pure dl-api as is... ie it should maybe mostly be seen as an example.  I'm actually wondering if dl_util should just be replaced with some examples? 
For example I don't use dl_util in my own code as I want to add my own allocators etc...

Reply:
In what way does it now use the internals of "dl_load"? `dl_instance_load_inplace()` is public api, and the old usage of `dl_instance_load()` was "inplace" without actually calling `dl_instance_load_inplace()`.

Reply:
I wouldn't mind replacing utils with examples instead, but the change of memmove -> memcpy feels right